examples/validate_relative_binding.py

```diff
@@ -195,14 +194,16 @@ def fn(*args, **kwargs):
         client = GRPCClient(hosts=cmd_args.hosts)
     client.verify()
 
+    seed = cmd_args.seed
+    if seed == 0:
```

Comment:
This seems like a pretty common convention in our code and I feel I have seen it other places. But can make it None

Reply:
If we're changing this convention I prefer the type-based "`seed: Optional[int]`" over the flag-based "`seed: int`, see docstring for flag value".

The convention here may be intentional. (The C++ / CUDA implementations use `seed == 0` as a flag to randomize the seed -- it may be valuable to have reminders of this also in the Python layer.)

Anyway, may be best to defer to another Issue / PR.

>0 for the seed will result in a random seed (admittedly that's probably not a common value to pass)

Hah, personally, I had used `np.random.seed(0)` more commonly than any other value, until switching to the `np.random.seed(2022)` convention.